crates/primitives/src/chain/spec.rs

```diff
@@ -243,6 +243,56 @@ pub static DEV: Lazy<Arc<ChainSpec>> = Lazy::new(|| {
     .into()
 });
 
+/// The Optimism Mainnet spec
+#[cfg(feature = "optimism")]
```

Comment:
this should mention that this does not include the genesis state, no alloc field in genesis file